snippets/header-drawer.liquid

```diff
@@ -109,7 +109,14 @@
                 </li>
               {%- endfor -%}
               <li>
-                <a href="/collections/shop-now" class="menu-drawer__menu-item list-menu__item link link--text focus-inset">
+        
+                {% if customer.tags contains 'wholesale' %}
+                  {% assign shopLink = '/collections/wholesale' %}
```

Comment:
update link here too

Reply:
ok but there should only be two badge colors: "sale" and "sold out" 
parter price and sale is really the same badge (different text)

Reply:
I said no to the red color. Only two colors. Partner and Sale are both the green color.